torch/csrc/jit/tracer.cpp

```diff
@@ -192,7 +192,10 @@ void addInputs(Node* n, const char* name, const ArrayRef<double>& value) {
 }
 
 void addOutput(Node* node, const at::Tensor& output) {
-  Value* value = node->addOutput();
+  setOutput(node->addOutput(), output);
```

Comment:
why even have `addOutput` then?

Reply:
Pretty sure it is used in a bunch of codegen.